src/Auth0.ManagementApi/Clients/GuardianClient.cs

```diff
@@ -127,7 +128,7 @@ public Task<GuardianTwilioConfiguration> GetTwilioConfigurationAsync(Cancellatio
         /// <returns>The <see cref="UpdateGuardianFactorResponse" /> indicating the status of the factor.</returns>
         public Task<UpdateGuardianFactorResponse> UpdateFactorAsync(UpdateGuardianFactorRequest request, CancellationToken cancellationToken = default)
         {
-            var name = request.Factor == GuardianFactorName.PushNotifications ? "push-notification" : "sms";
+            var name = Utils.GetEnumMemberValue(request.Factor);
```

Comment:
Would `var name = request.Factor.ToEnumString();` work here instead of the new `GetEnumMemberValue` method?

Reply:
Yes good point, thanks!